test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,8 @@
 const { joinNames } = require("../src");
 
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+  test("returns string of names, seperated by commas and an ampersand", () => {
+    const inputNames = [{ name: "Bart" }, { name: "Lisa" }, { name: "Maggie" }];
+    expect(joinNames(inputNames)).toEqual("Bart, Lisa & Maggie");
+  });
```

Comment:
Should add more tests for edge cases. I would have separate test for when `inputNames` has a length of: 0, 1, 2, 3 and more than 3.

I have tested for each of these scenarios and your code works as expected. You should still create these test to confirm that and in case someone wants to go back and alter this function later, they can then be sure that the functionality hasn't been broken and get a better understanding of how this function should work with different inputs.